### ISSUE TITLE
[BUG] Search Panel No Longer Saves Width

### ISSUE THREAD

**Issue by Kanegasi** [notepad-plus-plus/notepad-plus-plus#15512]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description of the Issue

The find/replace window does not save its width between sessions.

### Steps To Reproduce

1. Open Notepad++, open search, mess with position and width, then close search
2. Open search again, verifying position and width
3. Close Notepad++ and reopen, open search

### Current Behavior

Top left position of search was saved, width was not

### Expected Behavior

Width of search to save

### Debug Information

```shell
Notepad++ v8.6.8   (64-bit)
Build time : Jun  4 2024 - 00:30:00
Path : C:\Program Files\Notepad++\notepad++.exe
Command Line : 
Admin mode : OFF
Local Conf mode : OFF
Cloud Config : OFF
Periodic Backup : OFF
OS Name : Windows 10 Pro (64-bit)
OS Version : 22H2
OS Build : 19045.4717
Current ANSI codepage : 1252
Plugins : 
    ComparePlugin (2.0.2)
    mimeTools (3.1)
    NppConverter (4.6)
    NppExec (0.8.8)
    NppExport (0.4)
    NppTextFX (1.4.1)
```

**Comment by alankilborn** [comment]
Cannot replicate, in 8.6.9

**Comment by Kanegasi** [comment]
Can confirm, width is correctly saved in v8.6.9. My apologies for not updating before submitting.

However, I noticed another issue related to my task bar, which is along the left side of the desktop. I believe I've had this issue for a very long time, but I just now realized it's not Notepad++ wanting its search window closer to the main window.

Every time Notepad++ closes and reopens, the search window's saved position moves to the left the width of the task bar. If I want the search window in the top right of my desktop, it will eventually touch my task bar after I close and reopen Notepad++ about 15 times.
